### PR TITLE
Bug fixes

### LINKED ISSUE #37275: Space Ninja tech disk functionality is broken
How to reproduce: As a Space Ninja, try using a tech disk with research nodes in it on your suit. It's supposed to upload the research, but instead it prints the message "No research information detected." Presumably this is just something that was missed when techwebs was added?

### LINKED ISSUE #45071: Mech teleporters can teleport directly to Centcom via cameras
Title, mech teleport equipment modules can cross z-levels while using them through cameras, allowing access to Centcom.

Did this once a while back after okaying it with admins during a long extended shift. I've forgotten the round ID however, so none is available.

## Reproduction:
- Play on metastation
- Wait until RnD researches exosuit teleportation modules
- Build a mech and attach a teleport module to it
- Go to the departures security checkpoint and stand next to the security monitor on the wall
- Eject from the mech, start climbing back into the mech and use the monitor before you finish climbing in (Entertainment/TV screens don't let you view them from inside a mech)
- Select any camera to gain vision of the Thunderdome and Centcom z-level
- Select the mech teleporter module, face east (or rotate randomly), and click anywhere you have vision.
- End up in the thunderdome, ~~interrupt~~ 'steal' whatever ~~ERP~~ 'fun' the admins were up to, and probably get something like [this](https://i.imgur.com/Lrzogne.png) shortly later

Security monitor used for this likely has the wrong network ID listed in its variables.
oversight

A camera built on the cargo shuttle also works as a destination with the use of regular security camera monitors.
Untested but camera bugs with this method in might might let you rapidly teleport about nearly anywhere where there's an accessible camera.

> Really the exact nature of what makes bluespace effective for teleportation remains a mystery, the guidebook on it being a parallel universe of proportionate size to ours is one thing, the blue radiation that likely gives it its name is another.
The only worthwhile lead on this would be some bluespace scientist by the name of Hilbert who was undoubtedly the most far along at researching bluespace before his disappearance.
The only other speculation I can provide is just a possible relation to the Doppler blueshift effect and anomalous region Space Station 13 was built in.
Whatever's here Nanotrasen isn't the only one interested in it, and I'm not just talking about plasma.

### LINKED ISSUE #47219: phobias can cause abductors to speak out loud
https://cdn.discordapp.com/attachments/326826573951008769/635045676325142528/cowardlyabductor.PNG

I had the phobia (supernatural) trauma and the brain damaged language brain trauma as an abductor, and being afraid of my own jumpsuit occasionally forced me to say "AHHHH!!!" out loud (in the brain damaged language) instead of in abductor chat.

Also, according to my experience in this round, you can apparently still be afraid of stuff while wearing a blindfold, although maybe that's because I was wearing the thing that I was afraid of (my own jumpsuit; thanks, random brain trauma event, very cool).

### LINKED ISSUE #48160: Monkey Cube Cloning
## Reproduction:If you use a dropper of water on a monkey cube you get both a monkey and another monkey cube that can be activated using a beaker of water making 2 monkeys.
Monkey Cube Cloning The following image is from one box of monkey cubes

### LINKED ISSUE #49974: Can try to tackle items you're wearing
## Round ID:
**Reporting client version:** 513.1513

**Server-Side Code Modifications:** https://tgstation13.org/parsed-logs/basil/server_side_modifications.dm

**Round ID:** [132255](https://scrubby.melonmesa.com/round/132255) 
**BYOND Version:** 513.1508
**Master commit:** 6a599a213b447d2b063d9b7e3e4f98955e2a30b1

**Local commit:** 14e141a5991084ebd09afca63e5cf9ac4bdab13e
**Server tools version:** /tg/station 13 Server v3.2.6.2

## Testmerges:
**#49344:** '[READY] Space Dragon Rework' by IndieanaJones at commit 8416daa7c4

## Reproduction:

1. Equip improvised gripper gloves
2. Turn on throw
3. Click one of the items in your slots
4. Fall over

> You leap at the improvised gripper gloves!

#49476
@Ryll-Ryll

### PR DESCRIPTION
**Pull request by Cruix**
Fixed some bugs. April fools.

Fixes #47219
Fixes #48160
Fixes #45071
Fixes #37275
Fixes #49974

Also fixed a runtime when a mob without a client attempts to point at something, such as due to a phobia reaction.

## Changelog
:cl: Cruix
fix: Fixed some phobia reactions forcing people with abductor tongues to speak out loud
fix: Causing a monkey powder reaction in a monkey cube now deletes the monkey cube
fix: Mech teleporters can now only select a destination within 7 tiles and on the same Z-level
fix: Space ninjas can now properly extract research data from technology disks
fix: You can no longer attempt to tackle items that are in your inventory
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
RIP teleporting to centcom

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -437,7 +437,9 @@
 	set name = "Point To"
 	set category = "Object"
 
-	if(!src || !isturf(src.loc) || !(A in view(client.view, src)))
+	if(client && !(A in view(client.view, src)))
+		return FALSE
+	if(!src || !isturf(src.loc))
```
Comment (spookydonut) [inline code/modules/mob/mob.dm:442]: this should be before the other check

File: code/modules/surgery/organs/tongue.dm [hunk 2]
```diff
@@ -137,7 +137,7 @@
 /obj/item/organ/tongue/abductor/handle_speech(datum/source, list/speech_args)
```
Comment (spookydonut) [inline code/modules/surgery/organs/tongue.dm:137]:
```suggestion
/obj/item/organ/tongue/abductor/handle_speech(mob/user, list/speech_args)
```
the signal is sent from /mob so its always a mob

File: code/modules/surgery/organs/tongue.dm [hunk 3]
```diff
@@ -137,7 +137,7 @@
...
 	//Hacks
 	var/message = speech_args[SPEECH_MESSAGE]
-	var/mob/living/carbon/human/user = usr
+	var/mob/living/carbon/human/user = source
```
Comment (spookydonut) [inline code/modules/surgery/organs/tongue.dm:140]:
```suggestion
```

**Review by spookydonut — CHANGES_REQUESTED** [review]

File: code/modules/surgery/organs/tongue.dm (hunk 2 above)
Comment (Cruix) [reply code/modules/surgery/organs/tongue.dm:137]: All other tongues do it with the typecast, and changing them all is out of scope for this PR.

**Review by SpaceManiac — APPROVED** [review]

**Comment by cacogen** [comment]
awesome

**Pull request merged**